### PR TITLE
(WIP) [deployment] terraform scripts for deploying pulsar cluster to microsoft azure

### PR DESCRIPTION
**Pull request by sijie**

### CODE REVIEW

File: deployment/terraform/templates/pulsar.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
```
Comment (rdhabalia) [inline deployment/terraform/templates/pulsar.sh]: this empty script is intentionally present?

Comment (sijie) [reply deployment/terraform/templates/pulsar.sh]: @rdhabalia this is still wip :) I am placing a empty script and updating the init script one by one

File: deployment/terraform/packer/variables.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "pulsar_version": "2.1.0-incubating",
```
Comment (rdhabalia) [inline deployment/terraform/packer/variables.json]: is it possible to not keeping it hard-coded?

**Comment by sijie** [comment]
> do you have plan to write document or cookbook to how to use this script?

yes will add documentation

**Pull request closed without merging**